starwars/src/components/AllChars.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import EachChar from './EachChar';
+import './StarWars.css';
+
+
+/**Take IN props.charslist
+ * <EachChar /> should be using {props.info}
+ */
+
+
+const AllChars = (props) => {
```

Comment:
The naming conventions `AllChars` and `EachChar` can be substituted with more semantic names like `CharacterList` and `CharacterCard`